src/resources/js/v2/tickets-commerce.js

```diff
@@ -84,6 +88,37 @@ tribe.tickets.commerce = {};
 		$document.on( obj.customEvents.hideLoader, obj.loaderHide );
 	}
 
+	/**
+	 * Display the notice component.
+	 *
+	 * @since TBD
+	 *
+	 * @param {Object} data Data for notice.
+	 */
+	obj.showNotice = function( data = {} ) {
+		const $notice  = $( obj.selectors.noticeItem );
+		const $content = $notice.find( obj.selectors.noticeContent );
+		const $title   = $notice.find( obj.selectors.noticeTitle );
+
+		let title   = data.title ?? tecTicketsCommerceData.default_notice.title;
+		let message = data.message ?? tecTicketsCommerceData.default_notice.content;
```

Comment:
No need for `title` and `message` to be defined by `let`, they can be `const`. Also no need to align the `=` signs.